client/e2e/workflows/bceidbusiness/industry_user_admin.spec.ts

```diff
@@ -0,0 +1,234 @@
+  // 👤 run test using the storageState for this role
+  const storageState = JSON.parse(
+    process.env.E2E_INDUSTRY_USER_ADMIN_STORAGE as string,
+  );
+  // Note: specify storageState for each test file or test group, instead of setting it in the config. https://playwright.dev/docs/next/auth#reuse-signed-in-state
+  test.use({ storageState: storageState }); // this will error if no such file or directory
+  test("Test Redirect to Dashboard", async ({ page }) => {
+    // 🛸 Navigate to dashboard page
+    const dashboardPage = new DashboardPOM(page);
+    await dashboardPage.route();
+    // 🔍 Assert that the current URL ends with "(authenticated)/dashboard"
+    await dashboardPage.urlIsCorrect();
+    await dashboardPage.dashboardTilesAreVisibleIndustryAdmin();
+  });
+
+  test("Operators Tile view and edit operator workflow", async ({ page }) => {
+    // 🛸 Navigate to operators tile page
+    const dashboardPage = new DashboardPOM(page);
+    const operatorPage = new OperatorPOM(page);
+    await dashboardPage.route();
+    await dashboardPage.urlIsCorrect();
+    await dashboardPage.dashboardTilesAreVisibleIndustryAdmin();
+
+    // Click Operator tile and view the Operator form
+    await dashboardPage.clickOperatorsTileIndustry();
+    await operatorPage.urlIsCorrect();
+    await operatorPage.operatorViewIsCorrect();
+
+    // industry_user_admin is able to view read only user operator form
+    await operatorPage.operatorFormIsDisabled();
+
+    // industry_user_admin is able to edit the operator form
+    await operatorPage.clickEditInformation();
+    await operatorPage.editOperatorInformation();
+    await operatorPage.clickSaveAndReturn();
+
+    await page.waitForURL(dashboardPage.url);
+    // Verify that we have returned to the dashboard
+    await dashboardPage.urlIsCorrect();
+
+    await dashboardPage.dashboardTilesAreVisibleIndustryAdmin();
+  });
+
+  test("Operations Tile Add Operation workflow", async ({ page }) => {
+    // 🛸 Navigate to operations tile page
+    const dashboardPage = new DashboardPOM(page);
+    const operationPage = new OperationPOM(page);
+    const operationsPage = new OperationsPOM(page);
+    await dashboardPage.route();
+
+    // Click Operations tile and view the Operations form
+    await dashboardPage.clickOperationsTileIndustry();
+    await operationsPage.urlIsCorrect();
+    await operationsPage.operationsTableIsVisible();
+
+    // industry_admin is able to view operations table with the following columns
+    await tableColumnNamesAreCorrect(operationsPage.page, [
+      "BC GHG ID",
+      "Operation",
+      "Submission Date",
+      "BORO ID",
+      "Application Status",
+      "Action",
+    ]);
+
+    // industry_user_admin is able to click the Add Operation button
+    await operationsPage.clickAddOperationButton();
+
+    // Verify that we are on the operation detail page
+    await operationPage.operationFormIsVisible();
```

Comment:
I don't think we need any of the `IsVisible()` assertions in this test--if the form isn't visible, then we're not going to be able to fill it in the next line and the assertions and happo will fail